### PR TITLE
enforce 50 <= description length <= 160

### LINKED ISSUE #327: Enforce that metadata descriptions are not more than 160 characters
Relates to https://github.com/Qiskit/documentation/issues/131. It's bad for SEO to exceed 160 characters.

We should also check that the title != description, which is a common practice. And probably a minimum of 50 characters for the description.

Skip APIs since those are generated.

We should include general tips for how to write good SEO, such as the tips from https://github.com/Qiskit/documentation/issues/131. We could either link to resources like the GitHub issue or directly write the tips.

--

This should simply modify `checkMetadata.ts`. Similar to our check for missing metadata, we should batch all failing files into a single list and single error message.

### PR DESCRIPTION
**Pull request by kevinsung**
Fixes #327

### CODE REVIEW

**Comment by psschwei** [comment]
:+1: for docs/run/quantum-serverless.mdx

**Review by Eric-Arellano — APPROVED** [review]
Thanks! We'll want to make an exception for API docs but this can be fixed in my follow up work to start running the metadata checker on API docs.

The new descriptions look solid to me. Maybe wait for sign off from @javabster because she did the first pass at rewriting these descriptions last time.

**Review by abbycross — APPROVED** [review]
👍

File: docs/start/index.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Introduction
-description: Introduction to IBM Quantum documentation
+description: Introduction to IBM Quantum, Qiskit, and Qiskit Runtime.
```
Comment (javabster) [inline docs/start/index.mdx:3]: I think this one should still mention the word documentation right? The page itself mostly explains what the docs platform is more than anything else

Comment (kevinsung) [reply docs/start/index.mdx]: Done.

**Review by Eric-Arellano — APPROVED** [review]

**Pull request merged**